draft-ietf-openpgp-pqc.md

```diff
@@ -379,13 +379,13 @@ The ML-DSA + EdDSA signature consists of independent ML-DSA and EdDSA signatures
 The OpenPGP message format allows multiple signatures of a message, i.e. the attachment of multiple signature packets.
 
 An implementation MAY sign a message with a traditional key and a PQ(/T) key from the same sender.
-This ensures backwards compatibility due to {{I-D.ietf-openpgp-crypto-refresh}}, Section 5.2.5, since a legacy implementation without PQ(/T) support can fall back on the traditional signature.
+This ensures backwards compatibility due to [[RFC9580, Section 5.2.5]](https://www.rfc-editor.org/rfc/rfc9580#name-malformed-and-unknown-signa), since a legacy implementation without PQ(/T) support can fall back on the traditional signature.
```

Comment:
I am not entirely sure whether this way of rendering the reference is common practice. In the new rendering, there is no more a link to the reference section in the same document. This means that the references to RFC 9580 without a specific section point to the reference section, and the references to the RFC with a specific section point to the URL.  Does anyone know an example of an RFC which uses references in this way?

According to https://www.rfc-editor.org/styleguide/part2/ the new variant seems to be OK, as the prescription there only pertains to the rendering of the references-section, and not to how the references in the text have to look.

@TJ-91 @wussler Any opinions on this?

Reply:
Maybe I am being dense, but do not see the problem with this reference format

Reply:
Yeah, I think it'll be OK. It's just that then you don't jump to the reference section by following the link, which is the only format I have seen so far in RFCs. But in any case, if our format should turn out not be OK, someone will tell us at some point and we can still revert it. In any case it is more convenient to jump to the referenced section directly.

Reply:
Ok, seems like I can resolve this conversation.